tests/Feature/SmartCars/SmartcarsWebInterfaceTest.php

```diff
@@ -33,10 +32,40 @@ public function itRedirectsFromDashboardAsGuest()
     }
 
     /** @test * */
-    public function itLoadsTheDashboard()
+    public function itLoadsTheDashboardAndExerciseButton()
     {
         $this->actingAs($this->account, 'web')->get(route('fte.dashboard'))
-                ->assertSuccessful();
+                ->assertSuccessful()
+                ->assertSeeText('View All Exercises')
+                ->assertDontSee('No Flight Training Exercises are available at the moment.');
+    }
+
+    /** @test * */
+    public function itDoesNotShowExercisesButtonWhenNoneEnabled()
```

Comment:
I'd advise changing the conventions of these tests in inline with the other feature tests i.e. testCalumIsAngry